cmd/podsync/main.go

```diff
@@ -135,6 +135,26 @@ func main() {
 		log.WithError(err).Fatal("failed to open storage")
 	}
 
+	environmentVariableMap := map[model.Provider]string{
+		model.ProviderYoutube:    "YOUTUBE_API_KEY",
+		model.ProviderVimeo:      "VIMEO_API_KEY",
+		model.ProviderSoundcloud: "SOUNDCLOUD_API_KEY",
+		model.ProviderTwitch:     "TWITCH_API_KEY",
+	}
+
+	// Replace API keys from config with environment variables
+	for provider, environmentVariable := range environmentVariableMap {
+		val, ok := os.LookupEnv(environmentVariable)
```

Comment:
Since we support API key rotations, we should probably be able to specify multiple keys (for example split by space).